[IMP] sales: menuselection fix (#3282)
odoo/documentation: content/applications/sales/crm/pipeline/multi_sales_team.rst

Multiple sales teams
====================

Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
their own unique sales processes.

> Create a new Sales Team

Reviewer: Would use [sentence case](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#titles-and-headings) here, just like the H1.
```suggestion
Create a new sales team
```

=======================

> To create a new Sales Team, go to :menuselection:`CRM --> Configuration --> Sales Teams` then click
> :guilabel:`Create`.

Reviewer: `Sales Team` doesn't need to be capitalized here, as it was already introduced as a term in the overview and it's not a proper noun.
```suggestion
To create a new sales team in Odoo, go to :menuselection:`CRM --> Configuration --> Sales Teams`
then click :guilabel:`Create`.
```

On the creation page, set an :guilabel:`Email Alias` to automatically generate a lead/opportunity

> for this Sales Team every time a message is sent to that unique email address. Choose whether to

Reviewer: Lowercase
```suggestion
for this sales team every time a message is sent to that unique email address. Choose whether to
```

accept emails from :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or
:guilabel:`Followers Only`.

> Set a :guilabel:`Domain` to assign leads/opportunities to this Sales Team based on specific

Reviewer: ```suggestion
Set a :guilabel:`Domain` to assign leads/opportunities to this sales team based on specific
```

Multiple sales teams
====================

Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
their own unique sales processes.

Create a new sales team
=======================

To create a new sales team, go to :menuselection:`CRM --> Configuration --> Sales Teams`, then
click :guilabel:`Create`.

On the creation page, set an :guilabel:`Email Alias` to automatically generate a lead/opportunity
for this sales team every time a message is sent to that unique email address. Choose whether to
accept emails from :guilabel:`Everyone`, :guilabel:`Authenticated Partners`, or
:guilabel:`Followers Only`.

Set an :guilabel:`Invoicing Target` if this team has specific monthly revenue goals. Set a
:guilabel:`Domain` to assign leads/opportunities to this sales team based on specific filters, such
as country, language, or campaign.

.. image:: multi_sales_team/sales-team-creation.png
   :align: center
   :alt: Create a sales team in Odoo CRM.

Add members to a sales team
---------------------------

To add team members, click :guilabel:`Add` under the :guilabel:`Members` tab when editing the sales

> team's configuration page. Select a salesperson from the dropdown menu or create new salesperson.

Reviewer: Compound modifier gets a hyphen.
```suggestion
team's configuration page. Select a salesperson from the drop-down menu or create new salesperson.
```

Set a maximum number of leads that can be assigned to this salesperson in a 30-day period to ensure
that they do not overwork.

.. image:: multi_sales_team/add-a-salesperson.png
   :align: center
   :alt: Add a salesperson in Odoo CRM.

One person can be added as a team member or :guilabel:`Team Leader` to multiple sales teams,
allowing them to access all of the pipelines that they need to.

Sales team dashboard
====================

To view the sales team dashboard, go to :menuselection:`CRM --> Sales --> Teams`. Odoo users will
see any teams that they are a part of as dashboard tiles.

Each tile gives an overview of the sales team's open opportunities, quotations, sales orders, and
expected revenue, as well as a bar graph of new opportunities per week and an invoicing progress
bar.

> .. image:: multi_sales_team/sales-team-overview.png

Reviewer: Resize + compress

> Sales Team dashboard

Reviewer: ```suggestion
Sales team dashboard
```
